features/admin.server-configurations.v1/constants/governance-connector-constants.ts

```diff
@@ -75,26 +75,25 @@ export class GovernanceConnectorConstants {
         SMS_OTP_CODE_LENGTH_MIN_LENGTH: number;
         SMS_OTP_CODE_LENGTH_MIN_VALUE: number;
     } = {
-
-        EXPIRY_TIME_MAX_LENGTH: 5,
-        EXPIRY_TIME_MAX_VALUE: 10080,
-        EXPIRY_TIME_MIN_LENGTH: 1,
-        EXPIRY_TIME_MIN_VALUE: 1,
-        MAX_FAILED_ATTEMPT_COUNT_MAX_LENGTH: 2,
-        MAX_FAILED_ATTEMPT_COUNT_MAX_VALUE: 10,
-        MAX_FAILED_ATTEMPT_COUNT_MIN_LENGTH: 1,
-        MAX_FAILED_ATTEMPT_COUNT_MIN_VALUE: 1,
-        MAX_RESEND_COUNT_MAX_LENGTH: 1,
-        MAX_RESEND_COUNT_MAX_VALUE: 5,
-        MAX_RESEND_COUNT_MIN_LENGTH: 1,
-        MAX_RESEND_COUNT_MIN_VALUE: 1,
-        SMS_OTP_CODE_LENGTH_MAX_LENGTH: 2,
-        SMS_OTP_CODE_LENGTH_MAX_VALUE: 10,
-        SMS_OTP_CODE_LENGTH_MIN_LENGTH: 1,
-        SMS_OTP_CODE_LENGTH_MIN_VALUE: 4,
-        SMS_OTP_EXPIRY_TIME_MAX_LENGTH: 4,
-        SMS_OTP_EXPIRY_TIME_MAX_VALUE: 1440
-    };
+            EXPIRY_TIME_MAX_LENGTH: 5,
```

Comment:
We don't need these indentations. for some reason local eslint config shows incorrect suggestions. I'm currently looking into this.

Reply:
Fixed